docs/sdks/swift-sdk/integrate-with-okta.md

```diff
@@ -56,7 +56,7 @@ let session = ASWebAuthenticationSession(
 
  - Step 4: Redirect URL
 
-To complete the authentication flow, launch another `ASWebAuthenticationSession` using the `redirectURL` returned from a successful authenticate response. The authorization code and the state parameter are attached to this URL.
+A `redirectURL` is returned from a successful authenticate response. The authorization code and the state parameter are attached to this URL. You can exchange the code for an id token using Okta's Token Endpoint.
```

Comment:
Will update this description with "outer" and "inner" OAuth after getting example app working

Reply:
Updated!